src/AutoRest.CSharp.V3/Output/Builders/SerializationBuilder.cs

```diff
@@ -29,7 +29,7 @@ public static ObjectSerialization BuildObject(KnownMediaType mediaType, ObjectSc
             }
         }
 
-        public static ObjectSerialization Build(KnownMediaType mediaType, Schema schema, bool isNullable)
+        public static ObjectSerialization Build(KnownMediaType? mediaType, Schema schema, bool isNullable)
```

Comment:
Doesn't make sense for this to be null

Reply:
If there are no responses, there is no mediaType. We built a lot of the system assuming a response is required.

Reply:
If there is no response then we are we calling this method?

Reply:
Fixed.